test/e2e/cluster_config.sh

```diff
@@ -85,6 +85,15 @@ clusters:
     profile: worker-default
     min_size: 0
     max_size: 21
+  - discount_strategy: spot
+    instance_types: ["m4.large", "m5.large", "m5.xlarge", "m4.xlarge"]
```

Comment:
does it respect order? If so why we go with m4 first instead of m5?

Reply:
IIRC it's whatever has the least potential for disruption, I'm not sure if order matters. It also doesn't matter anyway, the node would run a pause container and a small shell script and then get cordoned.

Reply:
Got it. Ofc it does not matter for the process, just thinking that `m5`s are cheaper.

Reply:
but we can't be sure for spots anyway, just ignore.